util/docker/maketest.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# Runs tests and reports on them to github
+
+DIR=$(cd $(dirname $0) ; pwd -P)
+source ${DIR}/docker_common.sh
+
+# Get sha sum
+USER="${GH_USER:-georgeburdell}"
+SUCCESS=true
+LINK_PREFIX="https://circle-artifacts.com/gh/RoboJackets/robocup-software/$CIRCLE_BUILD_NUM/artifacts/0$CIRCLE_ARTIFACTS/"
+ARTIFACT_DIR="/tmp/build_artifacts"
+PENDING=false
+SHORTNAMES=( )
+
+start_pending() {
+    SHORTNAME="$1"
+    curl -u $USER:$TOKEN \
+        -X POST https://api.github.com/repos/robojackets/robocup-software/statuses/${SHA_SUM} \
+        -H "Content-Type: application/json" \
+        -d '{"state":"pending", "description": "This check is pending. Please wait.", "context": '"\"circle/$SHORTNAME\""', "target_url": "http://www.robojackets.org/"}'
+}
+
+# A function to run a command. Takes in a command name, shortname and description.
+ci_task() {
+    CMD="$1"
+    SHORTNAME="$2"
+    DESCRIPTION="$3"
+
+    SHORTNAMES+=("${SHORTNAME}")
+
+    if [ "$PENDING" = "true" ]; then
```

Comment:
Instead of this, it might be better to just check for PENDING below and only make calls to `ci_task` if it's not true.  That way this function doesn't have to know about the pending stuff.